client/my-sites/email/email-providers-in-depth-comparison/comparison-table/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { FunctionComponent } from 'react';
+import type { TranslateResult } from 'i18n-calypso';
+
+export type ProviderComparison = {
+	header: string;
+	tools: TranslateResult;
+	storage: TranslateResult;
+	importing: TranslateResult;
+	support: TranslateResult;
+	selectCallback: () => void;
+};
+
+type InDepthComparisonProps = {
+	comparisonObjects: ProviderComparison[];
```

Comment:
Generally speaking, it doesn't seem like a good idea to me to suffix the name of a variable with a type. That only gives information about the internal structure, something that is really easy to infer with TypeScript anyway. I would rename that to `emailProviders`; it's a bit generic but it plays well when passed to `<ComparisonTable />` (I'm not attached to this particular name though, if we can find a more appropriate one).